app-laptop/thinkfan/metadata.xml

```diff
@@ -1,11 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		Thinkfan is a simple, lightweight fan control program.
+		Originally designed specifically for IBM/Lenovo Thinkpads,
+		it now supports any kind of system via the sysfs hwmon interface.
+		It is designed to eat as little CPU power as possible.
+	</longdescription>
 	<use>
-		<flag name="atasmart">include libatasmart support to get disc temperature</flag>
+		<flag name="atasmart">Use <pkg>dev-libs/libatasmart</pkg> to read temperatures directly from hard disks.</flag>
```

Comment:
These descriptions don't usually have dots in the end.

Reply:
That's new for me. No one complained so far about this.